samples/go/photo-index/main_test.go

```diff
@@ -91,9 +114,32 @@ func (s *testSuite) TestWin() {
 	}
 
 	s.Equal(4, len(idx.ByTag))
-	for i := 1; i < 5; i++ {
+	for i := 0; i < 4; i++ {
```

Comment:
It should have been 0...4 in the first place. 1...5 worked because all it did was (a) omit testing 0, and (b) inadvertently test that there were no tags called "tag4" (which is covered by the len test on line 116).